src/main/java/com/hellosign/sdk/HelloSignClient.java

```diff
@@ -746,6 +745,36 @@ public File getFiles(String requestId, String format) throws HelloSignException
         return httpClient.withAuth(auth).withGetParam(PARAM_FILE_TYPE_URI, format).get(url).asFile(fileName);
     }
 
+    public String getFilesUrl(String requestId) throws HelloSignException {
+        return getFilesUrl(requestId, SignatureRequest.SIGREQ_FORMAT_PDF);
+    }
+
+    public String getFilesUrl(String requestId, String format) throws HelloSignException {
```

Comment:
Is there a need for this signature as the `file_type` parameter is optional when fetching the `files_url` value.

https://app.hellosign.com/api/reference#get_files
`Returns a PDF or ZIP file, or if get_url is set, a JSON object with a url to the file (PDFs only)`